### PR TITLE
Minor improvements of the sync script

### PR DESCRIPTION
**Pull request by sdepold**

### CODE REVIEW

File: sync-repositories.sh [hunk 1]
```diff
@@ -1,51 +1,50 @@
...
+# Clear generated directories
+rm -rf "$DOCS_DIR/v7"
+rm -rf "$API_DIR/v7"
 
-mv .typedoc-build ../../static/api/v7
+# Get sources
+git clone -b feature/typedoc --single-branch "$MAIN_REPO_URL" --depth 1 "$DOCS_DIR/v7"
 
-cd ../..
+pushd "$DOCS_DIR/v7"
```
Comment (ephys) [inline sync-repositories.sh:25]: I didn't know about pushd, very cool

**Review by ephys — APPROVED** [review]

File: sync-repositories.sh [hunk 2]
```diff
@@ -1,51 +1,50 @@
...
-yarn docs
-
-mv esdoc ../../static/api/v6
+# Get sources
+git clone -b v6 --single-branch "$MAIN_REPO_URL" --depth 1 .sequelize/v6
 
-cd ../..
+pushd "$DOCS_DIR/v6"
+yarn # Install branch deps & build
+yarn docs # Generate v7 typedocs
```
Comment (WikiRik) [inline sync-repositories.sh:45]: For when the file will be changed next, we should fix this copy paste error

Comment (sdepold) [reply sync-repositories.sh:45]: 🤦

Comment (sdepold) [reply sync-repositories.sh:45]: https://github.com/sequelize/website/blob/main/sync-repositories.sh

**Pull request merged**